slides-2022/intro.md

```diff
@@ -0,0 +1,193 @@
+name: intro
+
+class: cols, middle, main
+
+.fifty.main-left[
+# Interactive Front-end Development
+
+by Artem Halas and Stanislav Deviatykh
+]
+.fifty.main-right[.main-circle[].secondary-circle[]]
+
+---
+# About the course
+
+* Elective course intended to provide practical experience
+* Started in spring 2016 by
+  * Urmas Talimaa (Glia)
+  * Stenver Jerkku (eAgronom)
+* Continued by Urmas Talimaa
+* Now continued by us
+
+---
+# About the course
+
+* Student feedback says that the course is difficult
+* Extensive **individual** programming homework
+* Listeners are welcome, but their homework will not be reviewed
+
+You will need to write at least 1000 lines of code, **by yourself** to complete
+homework. If this is not what you signed up for, there is still a option to
+de-register.
+
+---
+class: about-us
+# About us
+
+.cols[
+.fifty[
+### Artem Halas
+- Software engineer in Pipedrive
+- Love cycling
+
+.full-image[![Artem Halas](assets/artem_halas.png)]
+]
+
+.fifty[
+### Stanislav Deviatykh
+- Software engineer in Qminder
```

Comment:
@stdevi please add info about yourself and add a picture

Reply:
Done ✅